test/XRP/issued-currency-client.test.ts

```diff
@@ -0,0 +1,73 @@
+import { assert } from 'chai'
+
+import { Utils, WalletFactory, XrplNetwork } from 'xpring-common-js'
+import IssuedCurrencyClient from '../../src/XRP/issued-currency-client'
+import {
+  FakeXRPNetworkClient,
+  FakeXRPNetworkClientResponses,
+} from './fakes/fake-xrp-network-client'
+import 'mocha'
+
+const fakeSucceedingNetworkClient = new FakeXRPNetworkClient()
+// const fakeErroringNetworkClient = new FakeXRPNetworkClient(
+//   FakeXRPNetworkClientResponses.defaultErrorResponses,
+// )
+const walletFactory = new WalletFactory(XrplNetwork.Test)
+
+describe('Default XRP Client', function (): void {
+  it('Dummy description', function (): void {
+    // GIVEN an IssuedCurrencyClient.
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingNetworkClient,
+      XrplNetwork.Test,
+    )
+
+    // WHEN the compiler is told to shut up THEN everything is fine.
+    issuedCurrencyClient.shutUpCompiler()
+  })
+  it('Enable Deposit Auth - successful response', async function (): Promise<
+    void
+  > {
+    // GIVEN a IssuedCurrencyClient with mocked networking that will return a successful hash for submitTransaction
+    const xrpClient = new IssuedCurrencyClient(
+      fakeSucceedingNetworkClient,
+      XrplNetwork.Test,
+    )
+
+    const wallet = (await walletFactory.generateRandomWallet())!.wallet
```

Comment:
In the unit tests, this is something that we can perform once in a `beforeAll` method since the generated account won't actually be manipulated in any way.